### PR TITLE
Update Grouping Content

### PR DESCRIPTION
**Pull request by PeloWriter**
In preparation for future functionality, we're moving the grouping content into SDK-specific content and product content. 

The SDK fingerprinting content now lives in Usage > SDK Fingerprinting. The data management content is now in product > Data Management > Issue Grouping. This also had the knock-on effect of removing a series of `include` folders that weren't really needed (as the product side doesn't currently support the use of `include` statements).

### CODE REVIEW

File: src/docs/product/data-management-settings/event-grouping/index.mdx [hunk 1]
```diff
@@ -4,14 +4,15 @@ sidebar_order: 0
 redirect_from:
   - /data-management/rollups/
   - /learn/rollups/
+  - /data-management/event-grouping/
```
Comment (jan-auer) [inline src/docs/product/data-management-settings/event-grouping/index.mdx:7]: Shouldn't this redirect from the platform pages, or is that what it does?

**Review by jan-auer — APPROVED** [review]
The document is called "Issue Grouping", so since we're moving it already, I'd also use `issue-grouping` in the URL rather than `event-grouping`.

File: src/includes/grouping/example/_default.mdx [hunk 2]
```diff
@@ -1,10 +0,0 @@
-```discover {tabTitle: Enhancement Rules}
-# mark all functions in the std namespace to be outside the app
-family:native stack.function:std::*       -app
```
[the remark this answers is not in the document]
Comment (PeloWriter) [reply src/includes/grouping/example/_default.mdx:3]: Currently, on the product side of the house, the include statements don't work as they do for platforms. For now, we can add more examples and hard code them. In future, with more tools support, we can enable platform-specific support/include statements on the product side of our content.

File: src/docs/product/data-management-settings/event-grouping/index.mdx (hunk 1 above)
Comment (PeloWriter) [reply src/docs/product/data-management-settings/event-grouping/index.mdx:7]: This is the location in which it used to reside.

**Pull request merged**